test/integration_test.py

```diff
@@ -417,20 +417,17 @@ def _get_entities_by_project(self, entity_type, project_short_name):
         filters = json.dumps({'project': {'is': [project_short_name]}})
         entities = []
         size = 100
-        params = dict(filters=filters, size=str(size))
+        url = f'{config.service_endpoint()}/index/{entity_type}?filters={filters}&size={size}'
```

Comment:
This hurts my eyes. You actually reintroduced the broken way of composing the URL. Think about why this doesn't generally work.

Reply:
Yeah this composition makes future extensions of additional parameters to be a pain. Moved to composing with furl()